docs/setup_help.md

```diff
@@ -0,0 +1,69 @@
+---
+id: setup_help
+title: Set Up Help
+---
+
+Checkout the following for additional information and help in the setup process for Honeycomb for all operating systems!
+
+
+### Honeycomb Prerequisites
+
+Honeycomb relies on Electron to package the cross-platform desktop applications.
+
+
+
+### Help With Installing [Electron Prerequisites](https://brown-ccv.github.io/honeycomb-docs/docs/quick_start#0-installing-prerequisites) 
+You will need to install Electron's pre-requisites below. You can find the full instructions on the [electron documentation](https://www.electronjs.org/docs/development/build-instructions-gn) for your specific OS.
+
+Listed below are the prerequisites for each operating system along with resources for manually installing them.
+#### Windows
+
+- Visual Studio:
+  Install the latest version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the Desktop Development for C++ Workflow.
+  To add the workflow, follow [these instructions](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160#:~:text=If%20you%20have%20Visual%20Studio,Then%2C%20choose%20Modify).
```

Comment:
This is only a dependency for writing Windows apps, do we want to make a note of that?

Reply:
Could you elaborate on this a bit, we do need this for running electron on Windows correct? I assumed since it was under the Windows subsection it would be unique to Windows installation only